.stylelintignore

```diff
@@ -0,0 +1 @@
+**/src/react-datepicker/**
```

Comment:
Can we make this a direct path ?
```
packages/terra-datapicker/src/react-data-picker/**
```

Reply:
Also, this this is specific to thie date-picker pacakge, can we move this .stylelintingore to the package directory level? This would clean up this script as well
```diff
- "lint:scss": "stylelint src/**/*.scss --ignore-path ../../.stylelintignore",
+ "lint:scss": "stylelint src/**/*.scss --ignore-path .stylelintignore",
```

Reply:
That we can't do. We have two ways of doing the lint. We do it at the individual package level, and also at the terra-framework level. Even if we have this file here, we would still need to have the stylelineignore file at the root, so I think to reduce duplication, it's better to have it up top.